railties/lib/rails/commands/test.rb

```diff
@@ -1,5 +1,5 @@
-require "rails/test_unit/runner"
+require "rails/test_unit/minitest_plugin"
 
 $: << File.expand_path("../../test", APP_PATH)
 
-Rails::TestRunner.run(ARGV)
+exit Minitest.run(ARGV)
```

Comment:
I felt a bit weird adding `exit` here, but `autorun` also exits with what `Minitest.run` returns. Besides these tests failed without it: https://travis-ci.org/rails/rails/jobs/62532010#L819-L837.